src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+/**
+ * @author dogmax296
+ * @link https://github.com/dogmax296
+ */
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is started its work!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped its work!");
+    }
+
```

Comment:
redundant empty line

Reply:
if I delete it check styles plugin shows errors, didn't you know?

Reply:
I got File does not end with a newline. [NewlineAtEndOfFile] error